### ISSUE TITLE
TypeError with Chef client 12.6

### ISSUE THREAD

**Issue by rlaveycal** [DataDog/chef-handler-datadog#77]
I get the following error when the handler is called in chef client 12.6 on Windows 2012R2

````
[2016-02-03T11:28:08+00:00] ERROR: Report handler Chef::Handler::Datadog raised #<TypeError: no implicit conversion of nil into Array>
[2016-02-03T11:28:08+00:00] ERROR: C:/opscode/chef/embedded/lib/ruby/gems/2.0.0/gems/chef-handler-datadog-0.7.0/lib/chef/handler/datadog_chef_tags.rb:46:in `|'
````

This seems to be caused by the node_tags method returning nil 
````
    @node.tags.map! { |tag| 'tag:' + tag } if @node.tags
````

then being fed into 
````
	@combined_host_tags = chef_env | chef_roles | chef_tags
````

Changing it to 
````
    @node.tags ? @node.tags.map! { |tag| 'tag:' + tag } : []
````

fixes the issue

**Comment by miketheman** [comment]
Thanks for reporting! Any chance of a PR?

**Comment by miketheman** [comment]
I had done extensive research in to the Chef Client's `tags` method, and it's always _supposed_ to return an Array.
It makes sense to guard against it misbehaving, especially since it's been reported before.

**Comment by miketheman** [comment]
Resolved via #78, released in 0.8.0. Thanks!
